02-basics.Rmd

```diff
@@ -132,13 +132,13 @@ ncol(crashes)
 The results of the previous commands tell us that the dataset has 3 rows and 2 columns.
 We will use larger datasets, with thousands of rows and tens of columns, in later chapters but for now it's good to 'start small' to understand the basics of R.
 
-## Subsetting by index or name
+## Subsetting by index or name CHANGE TITLE OR BE CLEARER WHEN YOU ARE SUBSETTING BY INDEX OR NAME IN THE TEXT BELOW
```

Comment:
Good point. How about just 'Subsetting'? 

Would

'Subsetting vectors' 

be any better?

Reply:
Would need to define vector which I mentioned below. If 2.3 was subsetting then would you merge it with 2.4 (which is also subsetting)?  I think you could keep as subsetting by index or name if you make it clearer in the text what 'index' and 'name' mean.

Reply:
ah looks like you have sorted it below!!!